pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -930,7 +930,50 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 			Expect(causes).To(HaveLen(1))
 			Expect(causes[0].Field).To(Equal("fake.domain.memory.guest"))
 		})
-		It("should allow guest memory which is between requests and limits", func() {
+		Context("with guest memory greater than requested memory", func() {
+			var vmi *v1.VirtualMachineInstance
+
+			BeforeEach(func() {
+				vmi = api.NewMinimalVMI("testvmi")
+				guestMemory := resource.MustParse("128Mi")
+
+				vmi.Spec.Domain.Resources.Requests = k8sv1.ResourceList{
+					k8sv1.ResourceMemory: resource.MustParse("64Mi"),
+				}
+				vmi.Spec.Domain.Memory = &v1.Memory{Guest: &guestMemory}
+			})
+
+			It("should be rejected when VM has VFIO device", func() {
+				kvConfig := kv.DeepCopy()
+				kvConfig.Spec.Configuration.DeveloperConfiguration.FeatureGates = []string{virtconfig.GPUGate}
+				kvConfig.Spec.Configuration.PermittedHostDevices = &v1.PermittedHostDevices{
+					PciHostDevices: []v1.PciHostDevice{
+						{
+							PCIVendorSelector: "DEAD:BEEF",
+							ResourceName:      "example.org/deadbeef",
+						},
+					},
+				}
+				testutils.UpdateFakeKubeVirtClusterConfig(kvInformer, kvConfig)
```

Comment:
This should be moved in the `BeforeEach` section. So that both tests runs in the same kv config.